packages/docsearch-css/src/button.css

```diff
@@ -62,11 +63,8 @@
 }
 
 @media (max-width: 750px) {
-  .DocSearch-Button-KeySeparator,
-  .DocSearch-Button-Key {
-    display: none;
-  }
-
+  .DocSearch-Button-Keys,
+  .DocSearch-Button-Key,
```

Comment:
That line shouldn't be necessary since we hide its container.

Reply:
fixed in https://github.com/algolia/docsearch/pull/1033/commits/bfa436a97e03f26310e9989847e1792c5f5a61ef